### DISCUSSION TITLE
How to get FPS in test phase？

### DISCUSSION THREAD

**Discussion by kirkzZ** [facebookresearch/detectron2#3930] [Q&A]
How to get FPS in test phase？

**Comment by zhanghang1989** [comment]
fps should be displayed in the stdout

**Comment by kirkzZ** [comment]
thanks,one of the output is:
[12/17 03:23:47] d2.evaluation.evaluator INFO: Inference done 4946/5000. Dataloading: 0.0012 s/iter. Inference: 0.0556 s/iter. Eval: 0.0167 s/iter. Total: 0.0735 s/iter. ETA=0:00:03
how to compute FPS

**Comment by zhanghang1989** [comment]
The inference time is `0.0556 s/iter` 

You can calculate `fps = batch_size / 0.0556`

**Comment by kirkzZ** [comment]
I set   --num-gpus 4 SOLVER.IMS_PER_BATCH 16
and when test:
start inference on 5000 batches,

inference time is0.0556 s/iter,

total inference time is 0:06:07.214419,

the fps=5000/367.2 ?

**Comment by GarabetTopalian** [comment]
Where does the 367.2 comes from?
